convert2rhel/actions/system_checks/package_updates.py

```diff
@@ -54,12 +62,15 @@ def run(self):
             # repositories, we use this to catch exceptions when verifying if there is any packages to update on the system.
             # Beware that the `RepoError` exception is based on the `pkgmanager` module and the message sent to the output
             # can differ depending if the code is running in RHEL7 (yum) or RHEL8 (dnf).
-            logger.warning(
-                "There was an error while checking whether the installed packages are up-to-date. Having an updated system is"
-                " an important prerequisite for a successful conversion. Consider verifyng the system is up to date manually"
-                " before proceeding with the conversion."
+            self.add_message(
+                level="WARNING",
+                id="PACKAGE_UP_TO_DATE_CHECK_FAIL",
+                message=(
+                    "There was an error while checking whether the installed packages are up-to-date. Having an updated system is"
+                    " an important prerequisite for a successful conversion. Consider verifyng the system is up to date manually"
+                    " before proceeding with the conversion. %s" % e
```

Comment:
Probably we are missing the `str(e)` here?

Reply:
I'll add that